### ISSUE TITLE
Variable Name for the Module Path

### ISSUE THREAD

**Issue by gaelcolas** [RamblingCookieMonster/BuildHelpers#30]
Hiya, 
I've hit an issue with BuildHelpers when doing a `Set-BuildEnvironment -VariableNamePrefix ''`, where the $Env:PSModulePath is overriden and I wonder if its current name _actually_ makes sense.
I know that this is the reason why the Prefix exist in the first place, but before I work around it (which is trivial but not _pretty_), I wonder if that should not be renamed (`PS` is sort of reserved prefix, I believe). 

```PowerShell
$ENV:BHPSModulePath     via Split-Path on BHPSModuleManifest`
```
Should this not be ModulePath instead, I can't see the benefit of the PS prefix here?
I see the benefit of adding $ModulePath to $PSModulePath, but not replacing $Env:PSModulePath with $Env:BHPSModulePath (if that makes sense).

Gael

**Comment by RamblingCookieMonster** [comment]
Yeah, I could also see how it might generate confusion regardless of whether there's a prefix in front of it, good call

Stuck at home without of coffee, I'll be useless for an hour or so until I can head out to grab some - any suggestions on an alternative name?

**Comment by gaelcolas** [comment]
No Worries it's Friday! 
If you're happy with `ModulePath` I can do a quick PR before I get out for dinner?

**Comment by RamblingCookieMonster** [comment]
Perfect!  Up to you, no rush : )
